### PR TITLE
Fix: No widget data error

### PR DESCRIPTION
**Pull request by migh**

### CODE REVIEW

File: src/components/widget/templates/mangrove-net-change/component.js [hunk 1]
```diff
@@ -91,27 +91,22 @@ class MangroveNetChange extends PureComponent {
     />);
 
     return (
-      <Fragment>
+      <>
```
Comment (mluena) [inline src/components/widget/templates/mangrove-net-change/component.js]:
```suggestion
```

File: src/components/widget/templates/mangrove-coverage/component.js [hunk 2]
```diff
@@ -60,65 +65,68 @@ class MangroveCoverage extends React.PureComponent {
...
+      const widgetData = this.getData();
+      const { percentage } = widgetData[0];
+      const unitOptions = [
+        { value: '%', label: '%' },
+        { value: 'km', label: 'Km' }
+      ];
+      const totalCoverage = metadata.total / 1000;
+      const coverage = (percentage * totalCoverage) / 100;
+      const quantity = numberFormat((unit === '%') ? percentage : coverage);
+      const location = (currentLocation.type === 'worldwide')
```
Comment (mluena) [inline src/components/widget/templates/mangrove-coverage/component.js]: is this working? 
```suggestion
      const location = (currentLocation.location_type === 'worldwide')
```

File: src/components/widget/templates/mangrove-coverage/component.js [hunk 3]
```diff
@@ -60,65 +65,68 @@ class MangroveCoverage extends React.PureComponent {
...
-            <Select
-              className="notranslate"
-              width="auto"
-              value={currentYear}
-              options={optionsYears}
-              onChange={this.changeYear}
-            />
-            {'.'}
+            <strong className="notranslate">{ quantity } {unitSelector}</strong><br />
+            <span>of</span> <strong>{ location }</strong>
```
Comment (mluena) [inline src/components/widget/templates/mangrove-coverage/component.js]:
```suggestion
            <span>of </span> <strong>{ location }</strong>
```

File: src/components/widget/templates/mangrove-coverage/component.js [hunk 4]
```diff
@@ -60,65 +65,68 @@ class MangroveCoverage extends React.PureComponent {
...
+            <strong className="notranslate">{ numberFormat(totalCoverage) } km</strong> coastline<br />
```
Comment (mluena) [inline src/components/widget/templates/mangrove-coverage/component.js]:
```suggestion
            <strong className="notranslate">{ numberFormat(totalCoverage) } km<sup>2</sup></strong> coastline<br />
```

File: src/components/widget/templates/mangrove-coverage/component.js [hunk 5]
```diff
@@ -60,65 +65,68 @@ class MangroveCoverage extends React.PureComponent {
...
 
-    return (
-      <Fragment>
-        <div className={styles.widget_template}>
+    try {
+      const widgetData = this.getData();
+      const { percentage } = widgetData[0];
+      const unitOptions = [
+        { value: '%', label: '%' },
+        { value: 'km', label: 'Km' }
```
Comment (mluena) [inline src/components/widget/templates/mangrove-coverage/component.js]: We need to change that label into km2 as well, we can use <DangerousHTML html={label} /> , we can do that in another task if you prefer

Comment (migh) [reply src/components/widget/templates/mangrove-coverage/component.js]: I think for this widget is linear quantity so km is fine... it is coastline, not mangrove area. We can confirm that with Ed or Lammer

**Pull request merged**